docs/cluster-configuration.md

```diff
@@ -22,21 +22,23 @@ cluster = Cluster(ClusterConfiguration(
     image="quay.io/project-codeflare/ray:latest-py39-cu118", # Mandatory Field
     instascale=False, # Default False
     machine_types=["m5.xlarge", "g4dn.xlarge"],
-    ingress_domain="example.com" # Default None, Mandatory for Kubernetes Clusters
+    ingress_domain="example.com" # Default None, Mandatory for Vanilla Kubernetes Clusters - ingress_domain is ignored on OpenShift Clusters as a route is created.
+    local_interactive=False, # Default False
 ))
 ```
+Temporary Note: On OpenShift, the `ingress_domain` is only required when `local_interactive` is enabled. - This may change soon.
```

Comment:
```suggestion
Note: On OpenShift, the `ingress_domain` is only required when `local_interactive` is enabled. - This may change soon.
```
Smallest nit